README-Client.txt

```diff
@@ -53,16 +53,19 @@ hubmap_api_py_client.Client = class ExternalClient(builtins.object)
  |      Returns:
  |          ResultsSet
  |  
- |  select_datasets lambda self, where=None, has=None
+ |  select_datasets lambda self, where=None, has=None, genomic_modality=None, logical_operator=None, min_cell_percentage=None
```

Comment:
`logical_operator` isn't documented... but maybe you've removed it? I remember suggesting that we minimize the feature space for now, so we can be more sure that what we have works as expected.

Reply:
I removed it from the documentation, I did mean to remove it elsewhere and will do that now.